lib/sqlalchemy/sql/operators.py

```diff
@@ -690,6 +690,36 @@ def ilike(
         """
         return self.operate(ilike_op, other, escape=escape)
 
+    def bitwise_xor(self, other: Any) -> ColumnOperators:
+        """Return bitwise XOR operation"""
+
+        return self.operate(bitwise_xor_op, other)
+
+    def bitwise_or(self, other: Any) -> ColumnOperators:
+        """Return bitwise OR operation"""
+
```

Comment:
**mike bayer** (zzzeek) wrote:

adding versionadded:: 2.0.2 to these

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4411